projects/Amlogic/kodi/appliance.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<settings version="1">
+
+  <section id="system">
+    <category id="display">
+      <group id="1">
+        <setting id="videoscreen.screenmode">
+          <visible>true</visible>
+          <default>0192001080060.00000pstd</default>
```

Comment:
why?

Reply:
Ping @chewitt

Reply:
because without it new installs auto-run at 4k screen resolution which leads to flashing screens and other bad things that result from the "not completely finished and tested" DRM driver. Once it's been finished up and 4k is stable it can be removed - although most devices will be more performant at 1080p so I'd still argue it's a sensible default.

Reply:
> because without it new installs auto-run at 4k screen resolution which leads to flashing screens and other bad things that result from the "not completely finished and tested" DRM driver. Once it's been finished up and 4k is stable it can be removed - although most devices will be more performant at 1080p so I'd still argue it's a sensible default.

Makes sense. Just don't forget about it because in the future we don't want to override the preferred mode from drm